executorlib/interactive/create.py

```
@@ -0,0 +1,198 @@
+        validate_max_workers as validate_max_workers_flux,
+    )
+except ImportError:
+    pass
+
+
+def create_executor(
+    max_workers: Optional[int] = None,
+    backend: str = "local",
+    max_cores: Optional[int] = None,
+    cache_directory: Optional[str] = None,
+    resource_dict: dict = {},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)